services/movie.py

```diff
@@ -1,11 +1,12 @@
 from django.db.models import QuerySet
-
+from django.db import transaction
 from db.models import Movie
 
 
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
 ) -> QuerySet:
```

Comment:
The function `get_movies` returns a `QuerySet`, which is a good practice for deferred evaluation and efficient querying.